packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -71,7 +115,7 @@ function BlockInspector() {
 		);
```

Comment:
Not a change from this PR but might be worth improving: the above `renderedBlockClientId` uses both `_selectedBlockClientId` and calls `getSelectedBlockClientId()` again. We could avoid calling the selector twice by just using `_selectedBlockClientId` instead.

Reply:
Nice, thanks!